projects/models.py

```diff
@@ -48,54 +65,55 @@ class Project(models.Model):
         related_name="projects_catalog",
         help_text="Catalog id that this project uses",
     )
-    IMPACT_LEVEL_CHOICES = [
-        ("low", "Low"),
-        ("moderate", "Moderate"),
-        ("high", "High"),
-        ("pii/phi", "PII or PHI"),
-    ]
+    catalog_version = models.CharField(
+        max_length=64,
+        default="CMS ARS 3.1",
+        blank=False,
+        help_text="The Catalog version, for example, NIST 800-53r5",
+    )
     impact_level = models.CharField(
```

Comment:
Should we add a default `impact_level` that should correspond with the default catalog?